### DISCUSSION TITLE
直屬文化

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1877] [General]
直屬文化是上大學通常會有的，一般而言會提供你考古、課本、和請你吃飯的學長姐，也可以說是帶我們認識系上的領路人，如果剛好學長姐很善良、熱心助人，或者是卷哥卷姐夯哥夯姐，基本上就是平常有扶老太太過馬路XD 我覺得至少學期剛開始的路會平坦一些

以我自己來說，我的學長姐直屬就是一個普通人，但是問他問題都會盡量回答我。不過很好笑的是，有時候我會給他考古或習題解答，變成是我救他哈哈哈

而我的學弟妹直屬就是完全不會來吵我，基本上跟消失了差不多，竟然都沒有問題要問，選課、怎麼歐趴也不問，雖然我是樂得清閒

很好奇直屬文化是怎麼發展出來的？還是只是過去的學長姐想聯誼而產生的XD 但不得不說，還是有幫忙到剛入學的學弟妹啦

不知道大家的直屬之路是怎樣的呢？因為有聽過被放生的、被騷擾的，千奇百怪的都有，很想知道新故事XDD

**Comment by FakeWood** [comment]
以前大一的時候會想說自己以後要當一個超扛的直屬，讓我的學弟妹能夠舒舒服服的無痛升大學，但有了直屬學弟妹以後才發現不知道要做甚麼，頂多就送個歐趴糖，請個家聚而已。

**Comment by yuchien-lin** [comment]
我跟我的學長直屬，除了在我微積分第一次會考有傳訊息說要來給我歐趴糖以外，就再也沒有任何的對話，他也沒有給我任何課本，基本上我跟他感覺就是陌生人，只是掛上了直屬這個名稱而已。

**Reply by vchien03** [reply]
跟我的經驗好像~真的就像掛名直屬

**Comment by RavenJiaSin** [comment]
我跟自己直屬的交流大概只停留在家聚和歐趴糖，反而是其他活動認識的學弟在互動上還比較像我的直屬。如果你的直屬是經由系上程序選的(抽籤/隨機配對)，感覺這種強制綁定的關係如果不去經營就只會變成有名無實了。

**Comment by vchien03** [comment]
我跟我的直屬學長只有在大一開學那陣子比較有往來，之後就形同陌路，沒有收到課本也沒有歐趴糖，而且因為他轉系，我們交集就更少了，甚至我的其他同學跟他還比較熟。我覺得真正會熟絡起來的，真的要性格契合，也要交集夠多、見面次數多，才會變比較熟。

**Comment by TobywithDino** [comment]
我跟我上面的直屬這學期根本沒見過面，去年也只看過兩次，一次是互認，一次是歐趴糖，超級難過，本來學長要請我吃家聚，結果我確診之後說好要延期，然後就不了了之了@@

**Comment by chuanmin0317** [comment]
我跟直屬學弟的互動次數也沒有很多，只有在家聚和送歐趴糖的時候有再見面過，我覺得目前關注到的直屬之間的關係都差不多是這樣，不過我覺得有在開學時學弟迷茫的時候幫助到他們就好了

**Comment by YifenLin** [comment]
我自己也有直屬學弟給我考古的情形，因為我的狀況比較特殊，我是轉系生。剛好我朋友轉學轉出我要轉入的系所，因此我就加進他們家，防止他家斷掉，我也可以得到考古資源嘿嘿。不過我跟直屬的互動也僅止於打招呼/要考古/給考古，感覺只像同學而非朋友，感覺生活圈重疊較多的話比較有機會成為好朋友的關係。

**Comment by KH1126** [comment]
大一的時候有四個直屬學姊可以讓我問問題，也在開學時給了我歐趴糖。
但大二開始後，因為轉系的關係，我就沒有直屬了。需要考古題或是要詢問問題，就是開始找原本不認識的學長姐詢問。

**Comment by 110101501** [comment]
我比較幸運，無論是大一屆的直屬，抑或是小一屆的直屬，相處下來都很愉快、興趣也蠻雷同。
我的直屬很照顧我，會提供我課本、考古等資源。同樣地，我也會以此方式對待我下一屆的直屬。不過自從轉系後就沒有認直屬了，當面臨到課堂上的問題時，大多則會詢問同屆的同學或是學長姊。

**Comment by jeff110303549** [comment]
直屬文化也算是大學生活裡比較有趣的一個傳統吧！我自己目前大三了，可以很驕傲地說我和直屬們關係都還不錯，雖然年輕的時候直屬不太給力，也沒幫到我太多忙，不過這不影響我盡量幫下面的直屬們，就是老話說：「淋過雨的人總想著幫別人撐傘」吧。

**Comment by sunset513** [comment]
我超級羨慕別人和直屬有良好的關係和互動，被照顧或是照顧其他人，感覺會很溫暖! 我自己是屬於被直屬放生的類型。我只有跟他講過一句話，就沒有然後了 QQ 。雖然被上面的直屬放生，但我有空的話還是會盡力照顧學弟妹們! 希望他們都能多受到一點照顧，感受到學長姐的人情 (不要像我一樣被放生)

**Comment by 20050201Shawn** [comment]
我跟直屬學長的互動蠻普通的，通常都是我問她課業上的問題，他也會正常答覆，平時沒事就不會有甚麼交集。今年就要當學姊了，心裡有點害怕自己的直屬弟妹會不會覺得我不是他們心中理想的樣子(像是可靠、有一堆考古題之類的)但又覺得自己不用先預設別人怎麼想，好好過好自己生活，有餘力幫助他們就好了。

**Reply by chia-kai-chang** [reply]
認同，自己先活得開心滿足，有餘力再幫助別人。真的不要給自己太大壓力。但是若有餘力，我們一起改變世界，讓世界更好❤️

**Comment by benbebobo149** [comment]
我是覺得直屬文化蠻棒的。或許是因為我很幸運，剛好我是其中一個受惠者。剛來到中央大學時，遇到了一些生活上的問題，我都會找我的直屬尋求幫助。相較於問同學或室友，問直屬更容易獲得有效的幫助，因為他們有更豐富的經驗和知識。這種直屬文化讓我感受到更多支持和幫助，也激勵我希望未來自己能成為像我的直屬一樣，能夠為新生提供支援和協助。感謝同學的分享，讓我有機會反思並分享我的感想。

**Comment by YANRONG0123** [comment]
我打從一開始，在直屬這個方面就已經毫無任何希望可言，因為我的直屬是轉校過來的，我也不會要求任何資源，但是，我對我的直屬就有提供相當多的考古題以及作業，確保他可以穩定做完基本的事情，目前也持續提供中。

**Comment by jylin34** [comment]
我覺得直屬文化很棒，因為我得到了有效的幫助和支持，如: 考古、各種時數取得管道等等，但之後由於轉系的緣故，導致很少跟直屬聯絡了，現在想起來也有點可惜。

**Comment by lyykey** [comment]
直屬文化在大學裡確實是一個很有趣的現象，每個人的經歷和感受都各不相同。對我而言，直屬文化既是一種幫助，也可能成為一種挑戰，謝謝同學分享。

**Comment by lyykey** [comment]
我的學長姐雖然不像卷哥卷姐那樣對待事事都充滿熱情，但他們都願意解答我的問題，這讓我感到很感激。有時候，我也能幫助他們，讓我覺得彼此之間的互助是很美好的。

**Comment by lyykey** [comment]
至於我的學弟妹，則是另一種情況。他們似乎不太需要我的幫助，或者是有自己解決問題的方式，這讓我有些意外也感到有些閒得發慌。雖然我很樂意幫助他們，但有時候也能享受一下清閒的時光。

**Comment by lyykey** [comment]
關於直屬文化的發展，我想可能是過去學長姐的一種自發性行為，希望能夠幫助新生適應大學生活。不過，隨著時間的推移，這種文化也許會隨著社會環境和學生的需求而有所變化。

**Comment by lyykey** [comment]
無論如何，直屬文化對於新生來說仍然是一個重要的支持系統，能夠在他們進入大學的新環境時提供幫助和指導。我期待著聽到更多有趣的直屬故事，無論是溫馨感人還是荒謬有趣，都讓人感到期待和好奇。

**Comment by wanderer657** [comment]
我還記得大學新生時的那段時光，對我來說，直屬學長張志明是那個特別的存在。他不僅是我的學術導師，更是我在大學裡的良師益友。當時的我充滿好奇和期待，而張志明的出現彷彿為我帶來了一道溫暖的光芒。

我第一次遇見張志明是在學校的迎新活動上。他親切地走到我面前，介紹自己並表示願意成為我的直屬學長。從那一刻起，他就像是一位領路人，帶領我進入了大學生活的世界。

張志明給我介紹了學校的各種設施和資源，幫助我選擇適合的課程，並分享了他在大學裡的學習和生活經驗。除了學業上的幫助，他還經常與我聊天，分享他的生活趣事和心得。

然而，時間飛逝，張志明畢業離開了大學。在告別的時刻，他對我說：“無論你走到哪裡，我都會永遠支持你。”這句話讓我感動不已，我知道在這段時間裡，他給了我無限的幫助和鼓勵。

即使他離開了，但張志明的影子仍然在我心中。現在，我已經長大了，我將帶著他的祝福和期許，勇敢地踏上自己的人生旅程。

**Comment by wanderer657** [comment]
大學的直屬文化真是有趣又奇怪。就像一個你從來沒見過的大哥大姐，帶你進入這個新的學術世界。有些直屬超熱心，像是考古、課本、還有時候請你吃飯的好朋友，簡直就是救世主！可我呢，我的學長姐們挺普通的，問她們問題她們會回答，不過有時候反過來，我竟然成了她們的考古救星，哈哈哈！

至於我的學弟妹們嘛，有些根本不理我，好像我是隱形人一樣。他們連選課、應該怎麼歐趴都不問，我倒也樂得清閒。不過，這種直屬文化是怎麼發展出來的呢？也許是過去的學長姐們想要多交點朋友，但不得不說，有時候這些直屬還是會給剛入學的學弟妹一些幫助。听說有些人的直屬之路是風平浪靜的，但也有些人卻被放生、被騷擾，真是千奇百怪啊！我超想知道更多新故事啊，哈哈哈！

**Comment by thomasxshen** [comment]
我認為直屬算是你剛進大學的引路人，有如紅髮之於魯夫、凱特之於小傑的關係一樣，能夠帶領你在茫然的大學生活找出一條屬於自己的路，當時我的學長給了我蠻多東西的，除了基本的考古題書本以外，還有告訴我可以去參加那些活動之類的，為我提供了不少選擇，然而他也如同紅髮一樣，留下書本考古題之後就不見縱影，我對於這件事情始終耿耿於懷。

**Comment by tzurou** [comment]
我自己和大我一屆的直屬就是平常不會聯絡，但我每學期考試前都會跑去要考古題，她也會把她有的東西都給我，家聚還會聊天不會太尷尬，是內向的好人! 我和下面的直屬就很熟，可能是因為我們都有加友會返服太常見面就變熟了，平常也會約吃飯出去玩，比較像是我嚮往中的直屬關係，我覺得自己蠻幸運的都遇到很好的直屬~

**Comment by 155Julia** [comment]
我的學姊直屬雖然有聯絡我也有讓我問問題，但她完全沒有請過我吃家聚，看著身邊的朋友都有免費家聚吃我真的很羨慕。雖然學姊有給我歐趴糖，但那竟然是過期的，不知道是不是從上面一直傳下來的，但我已經把它丟掉了，所以就先斷在我邊ㄏㄏ

**Comment by woohsin** [comment]
其實剛開學的時候很多事都不懂，直屬可以問是個不錯的機制啦(但也要看直屬有沒有要理你
而且吃家聚交流一下資訊也不錯

**Comment by willychiuchiu** [comment]
我覺得直屬文化很棒，但我是沒有直屬的可憐人嗚嗚，因為我的系的模式是學長姐自己去認大一當直屬，但因為當初剛入學就因為疫情遠距，沒有實體活動，都沒什麼機會遇到學長姐，導致很多人都沒找到直屬的情況。不過還好有朋友罩，不然連考古都沒有了

**Reply by 110101501** [reply]
還是你要現在認個直屬。

**Comment by happy246810** [comment]
我覺得有直屬很棒，但我沒有直屬啊!!!因為我們系是採取自己去找直屬學弟妹，所以如果不想要直屬，也可以不要，所以就產生我這種沒人要的邊緣人😭。所以我想說請把直屬文化變成一個規定，這樣就不會出現像我一樣的邊緣人了🤣。

**Comment by yyujimmy** [comment]
直屬文化的形成可能有多方面的原因。一方面，大學生活是一個新的開始，許多新生對於學校和系上的情況都不太了解，需要學長姐的指導和幫助。因此，學長姐願意主動幫助學弟妹，帶他們認識系上的環境和人事，也可以加強學長姐之間的聯繫和交流。

另一方面，直屬文化也可以促進學長姐與學弟妹之間的互助和交流。學長姐可以分享自己的學習經驗和知識，提供學習上的幫助和建議，同時也可以促進學弟妹之間的交流和合作。

不過，有時候直屬關係也可能會出現一些問題，例如學長姐不願意或沒有時間幫助學弟妹，或者學弟妹不願意接受學長姐的幫助，這可能會導致直屬關係的疏遠或不和諧。因此，建立一個良好的直屬關係需要雙方的努力和溝通。

**Comment by KH1126** [comment]
我自己和大我一屆的直屬平常不會聯絡，大一選課前都會去詢問關於哪個教授比較推薦的問題，他也會把他有的考古題給我。
轉系後，因此沒有直屬，所以每到考試都需要去詢問周遭朋友有無考古題可以支援。
所以有個好直屬可以解決很多問題，讚讚讚。

**Comment by William08290829** [comment]
我在大一系上沒有所謂的抽直屬，只有可能學長姊和學弟妹彼此心靈相通才會成家。而我就是那種落單的，這讓我每次考試都要問其他同學有沒有考古或是任何關於課程的資訊。我覺得直屬文化感覺還是要存在的，會讓大一新生比較有歸屬感的感覺。

**Comment by Eileen050824** [comment]
我自己對這個名詞感受很深，因為過去高中讀女校，存在於講求直屬制的儀式感文化中，我覺得是一種助力但某方面也是一種壓力。但我覺得既然已經上大學了，本就應該打理好自己，不要過度依賴學長姊資源，也不需要因為直屬的冷漠感到沮喪，好好經營自己的大學生活才是重點！

**Comment by wanka2030** [comment]
高中的時候因為有直屬而感到新奇，但最後才發現頂多就是兩三個月大家一起吃個飯，聊聊上學期都做了什麼，平常在學校也只是打個招呼、出去玩帶個伴手禮，但直屬也是一個可以讓你快速融入團體生活的一大契機！

**Comment by tshsu04224** [comment]
我的國中與高中生活也都有直屬文化，我覺得以我來說，這是一個很溫馨的文化，有種被特別照顧的感覺，在課業上或活動等地方有任何疑問，也有人可以為你解答，大學最重要的可能還是有考古吧，直屬的存在某部分來說也算是考古的傳承（？

**Comment by cassy0201** [comment]
我覺得直屬的角色對於開學的新生來說很重要，很多經驗和了開學階段準備事項都可以透過直屬去了解，但我發現大部分過了開學階段這份關係就會慢慢淡掉，平常沒事基本上都不太會打擾彼此，甚至根本沒見過幾次面

**Comment by hbh10601** [comment]
由於我有參加系學會的關係，對我來說和系學會的學長姐比較熟識一些，不過直屬送歐趴糖跟考古還有講義的時候真的挺讚的，大一徬徨的時候有人關心真的很幸福。

**Comment by YeeDaniel** [comment]
我們系上本身就有抽直屬的傳統，所以基本上每個人都會有直屬！還有我的直屬學長對我很好，在我大一時給了我很多的課本，讓我省了很多錢錢~ 還有給我一些考古，人真的超耐斯~

**Comment by yubo0oo** [comment]
直屬有跟你交流就相當不錯了
有給書或是給歐趴糖甚至關心你的都是天使
我的直屬這樣子對我所以我也這樣對我直屬

**Comment by poiu6fi** [comment]
我現在大二，整個大學生活看過我直屬的次數只有一次在開學的時候，平常沒有交集，他也不會出現在系上活動，好像整個人都神隱起來了，不過我自己是覺得沒差拉

**Comment by asd99263fg** [comment]
說真的我遇到的直屬其實還不錯，每次微積分的會考都有給我東西吃，講真的其實他大可不必對我有甚麼付出，但就是一個傳承吧，畢竟每個人都有事菜鳥的時候，所以有個可以帶領你到大學了解新的事務的引路人，其實會滿感謝的，至於要不要給東西吃那些的我道學的還好，主要還是傳達資訊必較重要
